<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" source-language="en" original="../Resources.resx">
    <body>
      <trans-unit id="DiscovererInstantiationException">
        <source>Exception occurred while instantiating discoverer : {0}</source>
        <note></note>
      </trans-unit>
      <trans-unit id="DuplicateAdaptersFound">
        <source>Multiple test adapters with the same uri '{0}' were found. Ignoring adapter '{1}'. Please uninstall the conflicting adapter(s) to avoid this warning.</source>
        <note>{0} is the unique identifier of test adapter. {1} is the name of the test adapter that shares a unique identifier with a previously loaded adapter.</note>
      </trans-unit>
      <trans-unit id="DuplicateSource">
        <source>Ignoring the specified duplicate source '{0}'.</source>
        <note></note>
      </trans-unit>
      <trans-unit id="ExceptionFromLoadTests">
        <source>An exception occurred while test discoverer '{0}' was loading tests. Exception: {1}</source>
        <note></note>
      </trans-unit>
      <trans-unit id="ExceptionFromRunTests">
        <source>An exception occurred while invoking executor '{0}': {1}</source>
        <note></note>
      </trans-unit>
      <trans-unit id="FileNotFound">
        <source>Could not find file {0}.</source>
        <note></note>
      </trans-unit>
      <trans-unit id="HostDebuggerWarning">
        <source>Host debugging is enabled. Please attach debugger to testhost process to continue.</source>
        <note>Host, testhost are key words, it should not be localized</note>
      </trans-unit>
      <trans-unit id="IgnoringExecutorAsNoDefaultExecutorUriAttribute">
        <source>Ignoring the test executor corresponding to test discoverer {0} because the discoverer does not have the DefaultExecutorUri attribute . You might need to re-install the test adapter add-in.</source>
        <note></note>
      </trans-unit>
      <trans-unit id="InitializationFailed">
        <source>Failed to initialize client proxy: could not connect to test process.</source>
        <note></note>
      </trans-unit>
      <trans-unit id="LaunchDebugProcessNotAllowedForANonDebugRun">
        <source>This operation is not allowed in the context of a non-debug run.</source>
        <note></note>
      </trans-unit>
      <trans-unit id="NoMatchingExecutor">
        <source>Could not find test executor with URI '{0}'.  Make sure that the test executor is installed and supports .net runtime version {1}.</source>
        <note>{0} - Executor uri String {1}-  Version - .net Version Eg:-2.0.50727.00</note>
      </trans-unit>
      <trans-unit id="NoValidSourceFoundForDiscovery">
        <source>None of the specified source(s) '{0}' is valid. Fix the above errors/warnings and then try again. </source>
        <note></note>
      </trans-unit>
      <trans-unit id="StringSeperator">
        <source>, </source>
        <note></note>
      </trans-unit>
      <trans-unit id="UnsupportedPropertiesInTestCaseFilter">
        <source>No tests matched the filter because it contains one or more properties that are not valid ({0}). Specify filter expression containing valid properties ({1}).</source>
        <note></note>
      </trans-unit>
      <trans-unit id="NoDotnetExeFound">
        <source>Could not find {0}. Make sure that the dotnet is installed on the machine.</source>
        <target state="new">Could not find {0}. Make sure that the dotnet is installed on the machine.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestHostExitedWithError">
        <source>Testhost process exited with error: {0}. Please check the diagnostic logs for more information.</source>
        <target state="new">Testhost process exited with error: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DataCollectorDebuggerWarning">
        <source>DataCollector debugging is enabled. Please attach debugger to datacollector process to continue.</source>
        <target state="new">DataCollector debugging is enabled. Please attach debugger to testhost process to continue.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestRunFailed_NoDiscovererFound_NoTestsAreAvailableInTheSources">
        <source>No test is available in {0}. Make sure that test discoverer &amp; executors are registered and platform &amp; framework version settings are appropriate and try again.</source>
        <target state="new">No test is available in {0}. Make sure that test discoverer &amp; executors are registered and platform &amp; framework version settings are appropriate and try again.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TesthostDiagLogOutputFile">
        <source>Logging TestHost Diagnostics in file: {0}</source>
        <target state="new">Logging TestHost Diagnostics in file: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="FailedToLaunchTestHost">
        <source>Failed to launch testhost with error: {0}</source>
        <target state="new">Failed to launch testhost with error: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="ExecutionThreadApartmentStateNotSupportedForFramework">
        <source>ExecutionThreadApartmentState option not supported for framework: {0}.</source>
        <target state="new">ExecutionThreadApartmentState option not supported for framework: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="OldTestHostIsGettingUsed">
        <source>You are using an older version of Microsoft.NET.Test.Sdk. Kindly move to a version equal or greater than 15.3.0.</source>
        <target state="new">You are using an older version of Microsoft.NET.Test.Sdk. Kindly move to a version equal or greater than 15.3.0</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NonExistingExtensions">
        <source>Could not find extensions: {0}</source>
        <target state="new">Could not find extensions: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DeprecatedAdapterPath">
        <source>Adapter lookup is being changed, please follow https://github.com/Microsoft/vstest-docs/blob/main/RFCs/0022-User-Specified-TestAdapter-Lookup.md#roadmap for more details.</source>
        <target state="new">Adapter lookup is being changed, please follow https://github.com/Microsoft/vstest-docs/blob/main/RFCs/0022-User-Specified-TestAdapter-Lookup.md#roadmap for more details.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoTestsAvailableForGivenTestCaseFilter">
        <source>No test matches the given testcase filter `{0}` in {1}</source>
        <target state="new">No test is available for testcase filter `{0}` in {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestDiscoveryCancelled">
        <source>Discovery of tests cancelled.</source>
        <target state="new">Discovery of tests cancelled.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="AttachDebuggerToDefaultTestHostFailure">
        <source>Cannot attach the debugger to the default test host with process ID: {0}.</source>
        <target state="new">Cannot attach the debugger to the default test host with process ID: {0}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="AccessDenied">
        <source>{0} Access denied while trying to create "TestResults" folder in mentioned location. You are getting this exception because you are running vstest.console.exe from a folder which requires having write access. To solve the issue: please  run vstest.console.exe from a folder where you have write privileges.</source>
        <target state="new">{0} Access denied while trying to create "TestResults" folder in mentioned location. You are getting this exception because you are running vstest.console.exe from a folder which requires having write access. To solve the issue: please  run vstest.console.exe from a folder where you have write privileges. For more information, please look at the error message:</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoAvailableProxyForDeque">
        <source>Could not find an available proxy to deque.</source>
        <target state="new">Could not find an available proxy to deque.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoSuchProxyId">
        <source>Proxy with id {0} is not managed by the current session manager.</source>
        <target state="new">Proxy with id {0} is not managed by the current session manager.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="ProxyIsAlreadyAvailable">
        <source>Proxy with id {0} is already available and cannot be re-enqueued.</source>
        <target state="new">Proxy with id {0} is already available and cannot be re-enqueued.</target>
        <note></note>
      </trans-unit>
    </body>
  </file>
</xliff>